cyipopt/scipy_interface.py

```diff
@@ -60,8 +60,6 @@ class IpoptProblemWrapper(object):
         for more information.
     eps : float, optional
         Epsilon used in finite differences.
-    n   : int, optional
-        Total number of variables.
```

Comment:
It isn't clear why this is removed.

Reply:
This should have been removed in [this commit](https://github.com/mechmotum/cyipopt/pull/121/commits/59bf2d19f68bf32756546693367e6256d521805e) in the last PR. Sorry for the mess!